rosdep/python.yaml

```
@@ -8226,6 +8226,14 @@ python3-smc-pip:
   ubuntu:
     pip:
       packages: [smc]
+python3-socketio:
+  debian:
+    bullseye: [python3-socketio]
+  fedora:
+    packages: [python3-socketio]
+  ubuntu:
+    focal: [python3-socketio]
+    jammy: [python3-socketio]
```

We should future-proof this syntax so that we don't need to revise the rules for the next Ubuntu release.

```suggestion
  ubuntu:
    '*': [python3-socketio]
    bionic: null
```